viadot/sources/databricks.py

```diff
@@ -428,3 +430,74 @@ def discover_schema(self, schema: str, table: str) -> dict:
         schema = dict(zip(col_names, data_types))
 
         return schema
+
+    def get_table_version(self, table: str, schema: str = None) -> numpy.int64:
```

Comment:
Is this really a numpy type and not a standard Python int? If so, can we make it a standard Python int? I think if we can have a simpler type, we should, and version number is a simple small number than can easily be stored as int. 

You probably have to convert it with int(your_numpy_number) but for me it's worth it when the brain doesn't have to stop and think "wait, why is a version number stored as a a numpy thingy?" 😛.